src/lakefs_spec/config.py

```diff
@@ -15,12 +15,15 @@ class LakectlConfig(NamedTuple):
 
     @classmethod
     def read(cls, path: str | Path) -> "LakectlConfig":
+        if not Path(path).exists():
+            raise FileNotFoundError(path)
```

Comment:
This is not covered by a unit test. Can you add a small one (not because that code is complex, but out of good habit ;))?